crates/pallets/starknet/src/tests/send_message.rs

```diff
@@ -0,0 +1,87 @@
+use frame_support::{assert_ok, bounded_vec};
+use mp_felt::Felt252Wrapper;
+use mp_transactions::compute_hash::ComputeTransactionHash;
+use mp_transactions::{DeclareTransactionV1, DeployAccountTransaction, InvokeTransactionV1};
+use starknet_api::transaction::TransactionHash;
+
+use super::mock::default_mock::*;
+use super::mock::*;
+use super::utils::get_contract_class;
+use crate::Config;
+
+// NoValidateAccount (Cairo 0)
+const DEPLOY_CONTRACT_SELECTOR: &str = "0x02730079d734ee55315f4f141eaed376bddd8c2133523d223a344c5604e0f7f8";
+
+// Generate: starkli class-hash ./cairo-contracts/build/send_message.json
+const SEND_MESSAGE_CLASS_HASH: &str = "0x024e12108da2b3d3c80c7bc99aacdec97e445199c5a8ded7435bf77ce4507631";
+// const SEND_MESSAGE_TO_L2_SELECTOR: &str =
+// "0x38fb0bf48fe489ae23d0a1d7f2b7195ec0b94bfeb2f408b13bfd943d8410d72";
+const SEND_MESSAGE_TO_L1_SELECTOR: &str = "0x9139dbd19ca9654d773cd88f31af4c8d583beecc3362fb986dccfef5cf134f";
+
+// Troubleshooting:
+// Add println! to the necessary runtime method (e.g. print CallInfo structs or extended blockifier
+// message) RUST_LOG=runtime=error RUST_BACKTRACE=1 cargo test --package pallet-starknet
```

Comment:
Why is `RUST_LOG=runtime=error` needed if you debug through `println`?

Reply:
Good question :)
I need to update the comment, it's just the notes for myself in order to to forget